message_ix/util/tutorial.py

```diff
@@ -3,17 +3,19 @@
 from functools import partial
 
 from message_ix import Scenario
-from message_ix.report import Key, Reporter, operator
+from message_ix.report import Key, Reporter, computations
```

Comment:
FYI I took this from @ywpratama's active PR. Were there recent changes to the core `Reporter` api? (including file name changes?)

I would not expect all of the change lines here based on what is in https://github.com/iiasa/message_ix/pull/806/files#diff-c26536ce84986d4689980ad63dd90f92b8012bdfcb3bd55ed3df380f71d0c56a

I'll note also this tutorial ran for me locally, so I'm confused if `message_ix.report` not longer has a `computations`. FYI @glatterf42

Reply:
Yes, there were changes in v3.8.0 to this exact module. See our [release notes](https://docs.messageix.org/en/stable/whatsnew.html#v3-8-0-2024-01-12). You should be using `from message_ix.report import operator` now instead of `... computations`.